framework/src/main/java/org/tron/core/services/jsonrpc/TronJsonRpc.java

```diff
@@ -292,9 +296,10 @@ String newFilter(FilterRequest fr) throws JsonRpcInvalidParamsException,
 
   @JsonRpcMethod("eth_newBlockFilter")
   @JsonRpcErrors({
+      @JsonRpcError(exception = JsonRpcInvalidRequestException.class, code = -32600, data = "{}"),
```

Comment:
According to [jsonrpc_specification](https://www.jsonrpc.org/specification), 
-32600 means "Invalid Request The JSON sent is not a valid Request object", but when the eth_newBlockFilter request exceeds the max num, the request itself is valid, so change to another code?